pages/linux/sar.md

```diff
@@ -0,0 +1,27 @@
+# sar
+
+> Monitor performance of various Linux subsystems.
+
+- Report I/O and transfer rate issued to physical devices, every second (press CTRL+C to quit):
+
+`sar -b {{1}}`
+
+- Report a total of 10 network device statistics, one per 2 seconds:
+
+`sar -n DEV {{2}} {{10}}`
+
+- Report CPU utilization every 2 seconds (press CTRL+C to quit):
```

Comment:
No need to mention 'press ctrl-c to quit' after the first time. It's obvious.

Reply:
sure